src/routes/MyProfile.js

```diff
@@ -1,7 +1,40 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
+import './MyProfile.css';
 
-const MyProfile = () => (
-  <div>MyProfile</div>
-);
+const MyProfile = () => {
+  const rockets = useSelector((state) => state.rockets.rockets);
+  const rocketsKey = Object.keys(rockets);
+  const reservedRocket = [];
+  rocketsKey.forEach((rocketId) => {
+    if (rockets[rocketId].reserved) {
+      reservedRocket.push({
+        id: rocketId,
+        name: rockets[rocketId].name,
+      });
+    }
+  });
+  return (
+    <div className="profile-display">
+      <div className="my-mission">
+        <h2>My Missions</h2>
+      </div>
+      <div className="my-rocket">
+        <h2>My Rockets</h2>
+        <div className="view-rockets">
+          {reservedRocket.length ? (
+            reservedRocket.map((rocket) => (
+              <tr key={rocket.id}>
+                <td>{rocket.name}</td>
+              </tr>
```

Comment:
-Kindly change the name of the labels to remove a warning that appears on console